tls/doc/tls_introduction.md

```diff
@@ -0,0 +1,263 @@
+# An Introduction to TLS
+
+Before we dive into the Exasol-specifics of TLS connections in the [next article](tls_with_exasol.md), let’s first discuss some TLS basics. We’ve created this introduction for people who want to get an idea about how TLS works. I am using metaphors, and I will simplify a lot of the details to make the article more accessible for everyone. If you’re already a cryptography expert, you can skip directly to ["TLS With Exasol"](tls_with_exasol.md).
+
+## What is TLS?
+
+Transport Layer Security — or TLS — is the successor to the Secure Socket Layer (SSL). While being a [separate standard](https://tools.ietf.org/html/rfc5246), people have become so used to saying 'SSL' that they often use it as a synonym. For the sake of precision, we are going to stick with TLS in this article.
+
+Simply put, TLS is meant to protect network communication against eavesdropping and offers a mechanism to verify that the other end of the connection is the one you intended to reach.
+
+Since people often have a wrong perception, I will stress here that a TLS connection does not protect against malicious intent of your communication partner.
```

Comment:
```suggestion
Since people often have an incorrect interpretation, I will stress here that a TLS connection does not protect against malicious intent of your communication partner.
```